src/ConfigInterface.php

```diff
@@ -155,6 +155,13 @@ public function setIndent($indent);
      */
     public function setLineEnding($lineEnding);
 
+    /**
+     * @param string $name
+     *
+     * @return self
+     */
+    public function setName($name);
```

Comment:
Adding a method to an interface is a BC break, please revert. As the constructor already allows passing a name, I don't think this method is useful anyway.

Reply:
RIght! Will remove it from Interface. :)

Reply:
As for usefulness... How often do you think people call "getIndent()" ? ;)
Yet it exists. If the method existed at first, this whole PR would be totally useless by now and I wouldn't be blocked by it. :)